### ISSUE TITLE
Fix focus when sidebar opens and closes

### ISSUE THREAD

**Issue by meganesu** [meganesu/generate-gb-opcodes#10]
Currently, when clicking on an instruction cell button, the focus remains on the button. Instead, it should move to the `h3` heading inside the sidebar.

Acceptance Criteria:

* WHEN I click on an instruction cell
THEN focus is moved to the heading (the instruction mnemonic) inside the sidebar
* WHEN I click on the close sidebar button
THEN focus is moved back to the instruction cell in the grid
* GIVEN my focus is inside the sidebar
WHEN I tab past the last tabbable element in the sidebar
THEN my focus should circle back to the first tabbable item in the sidebar (and not end up outside of the sidebar)

**Comment by meganesu** [comment]
- [ ] Change App to always render DetailsSidebar (with a class for when sidebar should be hidden - i.e., rendered offscreen)
- [ ] Move focus to DetailsSidebar heading when sidebar is opened
- [ ] Move focus back to instruction cell when sidebar is closed
- [ ] Trap focus inside the DetailsSidebar when it's open
